### PR TITLE
Update importing scenes for Godot 3.2 and move some stuff around

### PR DESCRIPTION
**Pull request by skyace65**
-Add FBX to the list of supported formats.
-Move Collada bellow gLTF since with newer versions of Blender people should use gLTF.
-Shorten gLTF description.
-Reword Collada description.
-Remove the section on FBX not being supported since support for FBX was added with the Open Asset Import Library.
-Move Collada Blender export instructions below gLTF since people should be using gLTF in newer versions of Blender.
-Explain that the better Collada export plugin doesn't work for Blender 2.8 or newer.

### CODE REVIEW

File: getting_started/workflow/assets/importing_scenes.rst [hunk 1]
```diff
@@ -13,26 +13,17 @@ transferred as close as possible.
 
 Godot supports the following 3D *scene file formats*:
 
-* DAE (COLLADA), which is currently the most mature workflow.
-* glTF 2.0. Both text and binary formats are supported. Godot has full support for it, but the format is new and gaining traction.
+* glTF 2.0. Godot has full support for text and binary formats.
+* DAE (COLLADA), an older format that is fully supported.
 * OBJ (Wavefront) formats. It is also fully supported, but pretty limited (no support for pivots, skeletons, etc).
 * ESCN, a Godot specific format that Blender can export with a plugin.
+* FBX, supported with the open asset import library. however it should only be used if you can't use another formet.
```
Comment (nezvers) [inline getting_started/workflow/assets/importing_scenes.rst]: Typo - ... if you can't use another formAt

**Comment by skyace65** [comment]
Fixed

**Comment by akien-mga** [comment]
Did some fixes (typos, Collada exporter will be ported to 2.80+ eventually so no need to document that).

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**